jenkins/common/sonic-wpa-supplicant-build-pr/Jenkinsfile

```diff
@@ -0,0 +1,33 @@
+pipeline {
+    agent { node { label 'sonic-slave-buster' } }
+
+    stages {
+        stage('Prepare') {
+            steps {
+                dir('sonic-wpa-supplicant') {
+                    checkout([$class: 'GitSCM',
+                          branches: [[name: '${sha1}']],
+                          userRemoteConfigs: [[url: 'http://github.com/Azure/sonic-wpa-supplicant',
+                                               refspec: '+refs/pull/*:refs/remotes/origin/pr/*']]])
+                }
+                dir('sonic-mgmt-common') {
+                    checkout([$class: 'GitSCM',
+                          branches: [[name: 'refs/heads/master']],
+                          userRemoteConfigs: [[url: 'http://github.com/Azure/sonic-mgmt-common']]])
```

Comment:
i do not think sonic-mgmt-common is needed.

Reply:
To be honest, I'm not sure.
According to my understanding, sonic-mgmt-common will be used to manager some configuration in the testing environment. So I included it.
But I will remove it according to your suggestion.